### PR TITLE
Add find-unicode-control2

### PR DESCRIPTION
**Pull request by massongit**
Fixes https://github.com/github/super-linter/issues/2088

## Proposed Changes
I add [find-unicode-control2](https://access.redhat.com/security/vulnerabilities/RHSB-2021-007#diagnostic-tools:).
It can find non-printable unicode characters and prevent the Trojan Source Attack ( https://trojansource.codes/ ).

### Author/Contributor
- [x] If documentation is needed for this change, has that been included in this pull request

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

**Comment by admiralAwkbar** [comment]
@massongit This is great, just need some time to review

File: Dockerfile [hunk 1]
```diff
@@ -38,6 +38,8 @@ FROM python:3.10.1-alpine as base_image
 ARG DART_VERSION='2.8.4'
 ## install alpine-pkg-glibc (glibc compatibility layer package for Alpine Linux)
 ARG GLIBC_VERSION='2.31-r0'
+# Unicode version info
+ARG UNICODE_VERSION='find_unicode_control2--2021-11-01-1136.zip'
```
Comment (massongit) [inline Dockerfile:42]: The version of find-unicode-control2 may be `2021-11-01-1136`.
Therefore, I think `UNICODE_VERSION` is `'2021-11-01-1136'`.

Comment (admiralAwkbar) [reply Dockerfile]: @massongit Good idea! updated!

**Comment by hpvd** [comment]
awesome!

**Pull request merged**